cluster/ubuntu/util.sh

```diff
@@ -107,7 +109,7 @@ function trap-add {
   trap "${handler}" ${signal}
```

Comment:
From shellchck.net: SC2064 (https://github.com/koalaman/shellcheck/wiki/SC2064) Use single quotes, otherwise this expands now rather than when signalled.